lib/src/layer/polygon_layer.dart

```diff
@@ -47,9 +51,19 @@ class Polygon {
     this.labelStyle = const TextStyle(),
     this.labelPlacement = PolygonLabelPlacement.centroid,
     this.rotateLabel = false,
-  }) : holeOffsetsList = null == holePointsList || holePointsList.isEmpty
-            ? null
-            : List.generate(holePointsList.length, (_) => []);
+  });
+
+  /// Used to batch draw calls to the canvas.
+  int get renderHashCode => Object.hash(
+      holePointsList?.length ?? 0,
```

Comment:
```suggestion
      holePointsList,
```

Since 2 lists with the same length would hash to the same value otherwise!

Reply:
Makes sense